src/main/java/org/spongepowered/common/data/processor/common/SkullUtils.java

```diff
@@ -57,7 +57,9 @@ public static boolean supportsObject(Object object) {
     }
 
     public static SkullType getSkullType(int skullType) {
-        return Iterables.get(SpongeImpl.getRegistry().getAllOf(SkullType.class), skullType);
+        return SpongeImpl.getRegistry().getAllOf(SkullType.class).stream()
```

Comment:
I almost want to say that I'd rather a traditional for loop to a stream, since streams are not quite yet optimized. Given the potential requirement of this being considered a "hot" method in some cases, it's best to be fast than to use fancy new API's, at least in the current implementation with streams.